api/db-cdr/generate-cdr/build-tanagra-static-prep-tables.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+set -e
+
+export BQ_PROJECT=$1        # project
+export BQ_DATASET=$2        # dataset
+
+################################################
+# CREATE STATIC PREP TABLES FOR TANAGRA
+################################################
+BUCKET="all-of-us-workbench-private-cloudsql"
+FOLDER="static_prep_tables"
```

Comment:
Are these used? Saw the Codacy failure

Reply:
Thanks for pointing out. Just pushed a fix.